packages/client-cli/lib/utils.mjs

```diff
@@ -45,3 +45,10 @@ export function getResponseContentType (responseObject) {
   const [firstContentType] = Object.entries(responseObject.content)
   return firstContentType[0]
 }
+
+export function is200JsonResponse (responses) {
+  if (responses['200'] && responses['200'].content && responses['200'].content['application/json']) {
```

Comment:
Other responses than `200` can also return `JSON` (Like a `fastify.error` for example). Maybe we just check `content`?

Reply:
Created https://github.com/platformatic/platformatic/issues/2197 related to this